bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/categorylist/FeaturedCategoryList.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+package com.adobe.cq.commerce.core.components.models.categorylist;
+
+import java.util.List;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+import com.adobe.cq.commerce.magento.graphql.CategoryInterface;
+
+@ProviderType
+public interface FeaturedCategoryList {
+
+    List<CategoryInterface> getCategories();
```

Comment:
The interface methods should have some JavaDoc

Reply:
Done